compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/statements/BLangClientDeclarationStatement.java

```diff
@@ -0,0 +1,71 @@
+/*
+*  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing,
+*  software distributed under the License is distributed on an
+*  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+*  KIND, either express or implied.  See the License for the
+*  specific language governing permissions and limitations
+*  under the License.
+*/
+package org.wso2.ballerinalang.compiler.tree.statements;
+
+import org.ballerinalang.model.tree.ClientDeclarationNode;
+import org.ballerinalang.model.tree.NodeKind;
+import org.ballerinalang.model.tree.statements.ClientDeclarationStatementNode;
+import org.wso2.ballerinalang.compiler.tree.BLangClientDeclaration;
+import org.wso2.ballerinalang.compiler.tree.BLangNodeAnalyzer;
+import org.wso2.ballerinalang.compiler.tree.BLangNodeTransformer;
+import org.wso2.ballerinalang.compiler.tree.BLangNodeVisitor;
+
+/**
+ * Represent a client declaration statement.
+ *
+ * @since 2201.3.0
+ */
+public class BLangClientDeclarationStatement extends BLangStatement implements ClientDeclarationStatementNode {
```

Comment:
We haven't used the term "Statement" in other places. "BLangClientDeclaration" should be enough I guess.

Reply:
It seems like we have used both. So current name should be fine.

Reply:
Yeah, used `BLangClientDeclaration` for the module-level declaration and `BLangClientDeclarationStatement` for the statement. Similar to `BLangXMLNS` and `BLangXMLNSStatement`. We can also change if required once the spec is updated.